jupyterhub_chameleon/utils.py

```diff
@@ -58,6 +58,7 @@ def from_query(cls, query):
 
     @staticmethod
     def chameleon_url_factory(deposition_id: str) -> str:
+        print(f"DEPOSITION {deposition_id}")
```

Comment:
Should use `logger`, if it logs at all

Reply:
An artifact of me deugging, left in by mistake